templates/default/rt/rtqueueviewbox.html

```diff
@@ -70,7 +70,9 @@
 		{elseif $queue[ticket].state == $smarty.const.RT_DEAD}
 		<span class="brown">
 		{/if}
-		{if $queue[ticket].deadline}{$queue[ticket].deadline|date_format:"%Y/%m/%d %H:%M"|default:"-"}{else}-{/if}<br>
+		{if $queue[ticket].deadline}{$queue[ticket].deadline|date_format:"%Y/%m/%d %H:%M"|default:"-"}
+		<br>{(($queue[ticket].deadline-$smarty.now)/86400)|round:"0"} {trans("days")}
```

Comment:
Z tym będzie w przyszłości problem jak deadline będzie z czasu letniego, a now z czasu zimowego (albo odwrotnie).
Liczba dni będzie wtedy o 1 przekłamana.

Reply:
Przypominam że od tego roku nie ma zmian czasu.

Reply:
A jak ktoś następny znowu to przywróci w ramach swojej "dobrej" zmiany?

Reply:
możesz spróbować przynajmniej zrobić **round(różnica między czasami)**. Dla polskiej strefy wtedy na pewno dobrze zadziała zawsze.

Reply:
@chillek takie zmiany są robione raz na kilkadziesiąt lat - szczerze wątpię.

Dorobię ten round.